CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ We use [semantic versioning](http://semver.org/):
 - PATCH version when you make backwards compatible bug fixes.
 
 # next release
+- [fix] _impacted-test-engine_: Handle tests with trailing whitespaces correctly
```

Comment:
```suggestion
- [fix] _impacted-test-engine_: Handle tests with trailing whitespaces correctly
- [breaking] _impacted-test-engine_: Now requires Teamscale 8.0 as minimum version
```

Maybe we should also document this minimum supported version in the README of the gradle and maven plugins, which is what users typically interact with